Gems/MotionMatching/Code/Source/FeatureMatrixStandardScaler.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <FeatureMatrix.h>
+#include <FeatureMatrixTransformer.h>
+
+namespace EMotionFX::MotionMatching
+{
+    //! The standard scaler can be used to normalize the feature matrix, the query vector and other data.
+    //! It standardizes features by removing the mean and scaling to the unit variance.
+    //! This implementation is mimicing the behavior of the standard scaler from scikit-learn (sklearn.preprocessing.StandardScaler).
+    //! As we use floats by default, our errors are a bigger, especially if the variance is small as this leads to a division by a small
+    //! value. In case the calculated standard deviation for a given feature is smaller than the given s_epsilon value, the standard
+    //! deviation gets force-set to 1.0 to avoid divisions by infinity and preserve the value when doing the transform -> inverse-transform
+    //! roundtrip.
+    class StandardScaler : public FeatureMatrixTransformer
+    {
+    public:
+        AZ_RTTI(StandardScaler, "{A0C7F056-94B0-43A1-8D12-B1A7B67AB92A}", FeatureMatrixTransformer);
+        AZ_CLASS_ALLOCATOR_DECL;
+
+        StandardScaler() = default;
+
+        bool Fit(const FeatureMatrix& featureMatrix, const Settings& settings = {}) override;
```

Comment:
suggestion: Api comment for this might be helpful?

Reply:
Is in the base class already

Reply:
Aah got you thanks 👍